### DISCUSSION TITLE
lllyasviel / IC-Light, looks amazing

### DISCUSSION THREAD

**Discussion by CantainPike** [lllyasviel/Fooocus#2888] [General]
IC-Light, Another great project by lllyasviel
https://github.com/lllyasviel/IC-Light
https://huggingface.co/spaces/lllyasviel/IC-Light

I played some on hugging face fantastic tool, I hope to see by  default in Fooocus .

Thank you llyasviel

**Comment by bsdesigner** [comment]
Of course, there is still work to be done, but the prospects for this technology are excellent:
![1](https://github.com/lllyasviel/Fooocus/assets/36730003/a07a1ceb-16cb-46a1-8201-8b624d840467)
![2](https://github.com/lllyasviel/Fooocus/assets/36730003/6bc0e730-aa57-4082-aa3a-1fbdff2118f3)
![3](https://github.com/lllyasviel/Fooocus/assets/36730003/82f37d3a-775b-4bf4-aa3c-6becc4b1dff8)

**Reply by mashb1t** [reply]
It also currently is built to support SD 1.5, imagine what it can do with SDXL or soon SD3!

**Reply by kyoushinosensei** [reply]
> It also currently is built to support SD 1.5, imagine what it can do with SDXL or soon SD3!

Please, answer here - is it worth waiting for it to be added IC-Light to Foocus?))
